src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -258,6 +267,15 @@ class DiscoverLanding extends AsyncComponent<Props, State> {
           placeholder={t('Search saved queries')}
           onSearch={this.handleSearchQuery}
         />
+        <PrebuiltSwitch>
+          <SwitchLabel>Show Prebuilt</SwitchLabel>
+          <Switch
+            isActive={renderPrebuilt}
+            isDisabled={renderPrebuilt && savedQueries.length === 0}
+            size="lg"
+            toggle={() => this.togglePrebuilt()}
```

Comment:
```suggestion
            toggle={this.togglePrebuilt}
```

`togglePrebuilt` is already using an arrow function so `this` will be correctly bound. Using another arrow function here can cause more re-renders than necessary.